src/sonic-config-engine/setup.py

```diff
@@ -15,7 +15,7 @@ def get_test_suite():
       author='Taoyu Li',
       author_email='taoyl@microsoft.com',
       url='https://github.com/Azure/sonic-buildimage',
-      py_modules=['minigraph', 'openconfig_acl'],
+      py_modules=['minigraph', 'openconfig_acl', 'sonic_platform'],
```

Comment:
I'm for splitting into multiple modules based on functionality. Actually I've always been feeling not comfortable to have port config logic in minigraph but have not had chance to refactor it.